### ISSUE TITLE
enh: Notify/Message all people in a group

### ISSUE THREAD

**Issue by stothew** [humhub/humhub#2048]
Would love to notify or message to groups like i do to users.
example:

@Designers
Have anybody time to correct this right now?

now the best would be: all people in the group "Designers" who are members of the Space i post get a notification.

Thanks! and keep up good work :)

**Comment by funkycram** [comment]
Yes, make `@all` or `@Designers` work as I use it every day in rocket.chat for example !

**Comment by buddh4** [comment]
Currently such a feature would not scale, for example if your group has thousands of users etc. In the future this could be solvable by using asnychronous jobs. Furthermore groups in humhub are rather meant to be administrative groups and are not involved in the social aspect of the network.

We also thought about a space chat, so you could just create a default space for your designer group and use the space chat.

**Comment by funkycram** [comment]
Thanks @buddh4 . The problem in a space is that if someone removes the notifications for this space in his settings (`Receive 'New Content' Notifications for the following spaces`) because he receives to many 'New Contents', I would like to have the possibility to notify @all members of a space if a new content is very important :
![image](https://user-images.githubusercontent.com/23310825/33542796-c7750ac2-d8d4-11e7-96af-147e8782d7c0.png)
Thanks,
Marc.

**Comment by acs-ferreira** [comment]
@funkycram see https://github.com/staxDB/humhub-modules-announcements

**Comment by funkycram** [comment]
Thanks @acs-ferreira . Yes, I forgot to come back to this issue, but your are right, this is a great module we use now all the time !

**Comment by funkycram** [comment]
This module does also the job: https://www.humhub.com/en/marketplace/mass_notification/
